containers/MainContainer.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { View, Text, Modal, Button } from "react-native";
+import Nav from '../views/Nav'
+import Game from '../views/Game'
+
+export default class MainContainer extends React.Component {
+
+  constructor() {
+    super()
+    this.state = {
+      navHidden: true,
+      playerDifficulty: {
+        playerOne: 1,
+        playerTwo: 3
+      }
+    }
+  }
+
+  toggleDifficulty = player => {

```

Comment:
Toggle generally refers to "on" or "off". This looks like it is cycling through 3 different difficulties. A better function name would be `setToNextDifficulty` or `cycleDifficulty` or something else along those lines

Reply:
Noted.